examples/bungalow-rental-and-usage.yaml

```diff
@@ -0,0 +1,164 @@
+        time:inXSDDateTimeStamp: 2019-02-05T00:00:00-5:00
+
+  # 10 more Commitments for MXN payments Mar - Dec
+
+  - '@id': elf:7dd207e3-8f11-4fe3-95f8-d02f6c180be3
+    '@type': EconomicEvent
+    under: notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    provider: tashirat
+    recipient: elf
+    action: lend
+    involves: tashirat:2402b57c-1c45-423a-acb2-f50961c97d65
+    observedQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 1
+    observedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-12-29T00:00:00-5:00
+
+  - '@id': elf:6b5bc786-b9ed-4189-b34f-5ef7d10f1f86
+    '@type': Fulfillment
+    under: notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    fulfilledBy: elf:7dd207e3-8f11-4fe3-95f8-d02f6c180be3
+    fulfills: tashirat:953b40f2-4e47-4486-afb2-402512f5269c
+    fulfilledQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 1
+    # TODO fulfilled duration
+
+  - '@id': elf:27be5cab-d740-4194-9298-1661a69d9d95
+    '@type': EconomicEvent
+    under: notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    provider: elf
+    recipient: tashirat
+    action: issue
+    resourceClassifiedAs: http://www.wikidata.org/entity/Q4730 # Mexican peso
+    observedQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 3000
+    observedTime:
+      '@type': time:Instant
+      time:inXSDDateTimeStamp: 2019-01-04T14:25:00-5:00
+
+  - '@id': elf:40df0092-5455-4507-83fc-7f3d64538163
+    '@type': Fulfillment
+    under: notary:40bb78fe-28df-41a6-b698-525fa9d411d6
+    fulfilledBy: elf:27be5cab-d740-4194-9298-1661a69d9d95
+    fulfills: tashirat:99bc3e4e-79a6-4b52-a178-8075a268f0df
+    fulfilledQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 3000
+
+
+  ## recording usage of bungalow
+
+  - '@id': elf:f12fc3b7-1033-4da0-85fd-a378b4cfb36f
+    '@type': EconomicEvent
+    skos:note: one month stay at the bungalow
+    action: use
+    affects: tashirat:2402b57c-1c45-423a-acb2-f50961c97d65
+    observedQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 1
+    observedTime:
+      '@type': time:ProperInterval
+      time:intervalStarts:
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2018-12-29T00:00:00-5:00
+      'time:intervalEnds':
+        '@type': time:Instant
+        time:inXSDDateTimeStamp: 2019-01-20T00:00:00-5:00
+
+  - '@id': elf:ee652454-ac08-4f5d-ae84-6b1753887086
+    '@type': Process
+    skos:note: a working session - 6 pomodoros
+
+  - '@id': elf:7a63ea10-b1c3-441a-9a08-fb8630c02614
+    '@type': EconomicEvent
+    skos:note: usage of bungalow as workspace durring one working session
+    action: use
+    affects: tashirat:2402b57c-1c45-423a-acb2-f50961c97d65
+    inputOf: elf:ee652454-ac08-4f5d-ae84-6b1753887086
+    includedIn: elf:f12fc3b7-1033-4da0-85fd-a378b4cfb36f
+    observedQuantity:
+      qudt:unit: unit:Unitless
+      qudt:value: 1
+    observedTime:
```

Comment:
Are you proposing changing affectedQuantity and affectedTime to observed...?  I think I might like observed better also.  But it means some changes to the handful of VF projects we have going now.  I think 2 of those will be finishing a major re-write in January, I'd like to do as much of this kind of naming cleanup as possible by say mid-January, because it will only get more painful.

Reply:
Oops, it is already observedTime